tracing-core/Cargo.toml

```diff
@@ -27,7 +27,7 @@ edition = "2018"
 rust-version = "1.42.0"
 
 [features]
-default = ["std"]
+default = ["std", "valuable/std"]
```

Comment:
this causes cargo to download the `valuable` crate even though it isn't actually used without `tracing_unstable`